mods/example/chrome/chrome.yaml

```diff
@@ -378,3 +378,15 @@ dropdown-separators:
 
 separator:
 	Inherits: button
+
+# TODO: Fix the random numbers below.
```

Comment:
Should be an easy fix. It is based on https://github.com/OpenRA/OpenRA/blob/release-20250330/mods/ra/uibits/glyphs.png minus the flags. You can also copy the coordinates.

Reply:
But these _are_ RA's values copied over.

P.S.: Oh, I see - the Example mod's `glyphs.png` is just missing those icons. Oh well, that's not my forte.

Reply:
You can just copy it with an image editor.